bpf/Makefile

```diff
@@ -81,7 +81,8 @@ LB_OPTIONS = \
 MAX_BASE_OPTIONS = -DSKIP_DEBUG=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 \
 	-DENABLE_HOST_SERVICES_TCP=1 -DENABLE_HOST_SERVICES_UDP=1 \
 	-DENABLE_HOST_REDIRECT=1 -DENABLE_ROUTING=1 -DNO_REDIRECT=1 \
-	-DPOLICY_VERDICT_NOTIFY=1 -DALLOW_ICMP_FRAG_NEEDED=1 -DENABLE_IDENTITY_MARK=1
+	-DPOLICY_VERDICT_NOTIFY=1 -DALLOW_ICMP_FRAG_NEEDED=1 -DENABLE_IDENTITY_MARK=1 \
+	-DMONITOR_AGGREGATION=3 -DCT_REPORT_FLAGS=0x0002
```

Comment:
Wouldn't an even lower aggregation level help to catch more problems?

Reply:
Good question :rocket: 

If we reduced `MONITOR_AGGREGATION`, we would have more trace point compiled in, with all of them compiled in if we reduce it to zero: https://github.com/cilium/cilium/blob/42eb1edafdaccad2814e7c16a1795e501fcd76b3/bpf/lib/trace.h#L130-L131

Usually I follow the heuristic of increasing program size in the hope it will increase complexity. So usually, I would have disabled monitor aggregation. However, in this case, if we set `MONITOR_AGGREGATION` to 2, the compiler won't be able to compile out this code:
https://github.com/cilium/cilium/blob/42eb1edafdaccad2814e7c16a1795e501fcd76b3/bpf/lib/trace.h#L144-L145
That in turn means we'll have one more conditional jump on the `monitor` variable whenever it's not constant. Each conditional jump is an opportunity to increase complexity.

I went for the second approach because recent a complexity issue (cf. https://github.com/cilium/cilium/issues/14552) revealed the impact of `MONITOR_AGGREGATION=2` on complexity may be higher than I had anticipated.

Reply:
:100: This would be the detail I overlooked when initially disabling monitor aggregation, Would be useful to include the detail in the git logs too.

Reply:
Makes sense. I've updated the commit message, rebased, and pushed.